src/main/pages/che-7/installation-guide/proc_adding-self-signed-ssl-certificates-to-che.adoc

```diff
@@ -0,0 +1,72 @@
+---
+title: Adding self-signed SSL certificates to Che
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/adding-self-signed-ssl-certificates-to-{prod-short}/
+folder: che-7/installation-guide
+summary:
+---
+:page-liquid:
+
+[id="adding-self-signed-ssl-certificates-to-{prod-short}_{context}"]
+= Adding self-signed SSL certificates to {prod-short}
+
+The following procedure describes how to configure {prod-short} to authorize HTTPS communication with various components, such as identity and git servers, by adding information about self-signed SSL certificates to a {prod-short} configuration.
+
+.Prerequisites
+
+* A Kubernetes orchestration tool, the OpenShift command-line tool, `oc`,
+ifeval::["{project-context}" == "che"]
+or the Kubernetes clusters controlling command-line tool, `kubectl`,
+endif::[]
+is installed.
+
+.Procedure
+
+. Save the desired self-signed certificates to a local file system.
+
+. Create a new *configMap* that will contain the desired self-signed SSL certificates:
++
+[subs="+attributes,+quotes"]
+----
+$ {orch-cli} create configmap __<configMap-name>__ --from-file=__<certificate-file-path>__ -n=__<che-namespace-name>__
+----
++
+To apply more than one certificate, add another `--from-file=_<certificate-file-path>_` option to the above command.
+
+. Define a name for the newly created configMap.
```

Comment:
```suggestion
. Define a name for the newly created *configMap*.
```

Reply:
why use the bold font in here?  :o It's not the part of the GUI

Reply:
You have it in bold a few lines above. :D Unless I'm misunderstanding, that one is also not referring to a UI element (and if I am, ingnore me ;D)) Let's just create consistency and not use bold at all, WDYT?

Reply:
Ooooh, I got it! Yeah, that's the residual of the original document .) yeah, I will put it away. Nice catch